ui/app/AppLayouts/CommunitiesPortal/panels/CommunityTagsPanel.qml

```diff
@@ -108,7 +121,7 @@ StatusScrollView {
             }
 
             StatusBaseText {
-                text: d.cntSelectedTags + "/" + maxSelectedTags
+                text: d.cntSelectedTags + " / " + maxSelectedTags
```

Comment:
And again, you really want to make this one string, doesn't even have to be translatable, ie `"%1 / %2"`, reason is that in RTL locales, it will be mirrored correctly